about.html

```diff
@@ -3,9 +3,23 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/index.css">
     <title>Document</title>
```

Comment:
This title in the head is what will show up on the browser tab as the title. You might want to change this to reflect what this page is. Aka About Alexandria Duell